### ISSUE TITLE
From 0.13.11 TypeError: object.__init__() takes no parameters

### ISSUE THREAD

**Issue by MihaiBalint** [tortoise/tortoise-orm#211]
**Describe the bug**
Received traceback:
```
ImportError while loading conftest '/builds/src/tests/conftest.py'.
tests/conftest.py:6: in <module>
    from .point_models import Point
tests/point_models.py:11: in <module>
    class Point(BaseModel):
tests/point_models.py:42: in Point
    null=False,
../.venv/lib/python3.6/site-packages/tortoise/fields.py:456: in __init__
    super().__init__(**kwargs)
../.venv/lib/python3.6/site-packages/tortoise/fields.py:92: in __init__
    super().__init__(**kwargs)  # type: ignore # mypy issue 4335
E   TypeError: object.__init__() takes no parameters
```

Excerpt from model definition:
```
class Point(BaseModel):
    policy_rules = fields.ManyToManyField(
        "models.PolicyRule",
        related_name="points",
        through="points_rules",
        on_delete=fields.CASCADE,
        null=False,
    )
```

This is a regression introduced in 0.13.11 (no traceback on 0.13.10)

**Comment by grigi** [comment]
Thank you for reporting the bug, I'll have a look at pushing a fix out asap. I have a good idea what it could be as there was only two changes for 0.13.11

**Comment by grigi** [comment]
Please test release v0.13.12 if it resolved the issue for you :-)

**Comment by MihaiBalint** [comment]
0.13.12 fixes this issue for me (tested with pythons 3.6.8 and 3.7.1)
